openfisca_us/variables/gov/irs/credits/cdcc/min_head_spouse_earned.py

```diff
@@ -0,0 +1,19 @@
+from openfisca_us.model_api import *
+
+
+class min_head_spouse_earned(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Less of head and spouse's earnings"
+    unit = USD
+    definition_period = YEAR
+
+    def formula(tax_unit, period, parameters):
+        is_joint = tax_unit("tax_unit_is_joint", period)
+        head_earnings = tax_unit("head_earned", period)
+        spouse_earnings = tax_unit("head_earned", period)
```

Comment:
Should this be spousal income?

```suggestion
        spouse_earnings = tax_unit("spouse_earned", period)
```

Reply:
Fixed and added a test to cover this